### PR TITLE
Initial OCE support

### PR DESCRIPTION
**Pull request by adamwolf**
Added OCE stuff per orsonmmz PR on the old nightly builder script.  Still in testing.

DO NOT MERGE.

### CODE REVIEW

File: kicad-mac-builder/CMakeLists.txt [hunk 1]
```diff
@@ -27,13 +27,11 @@ set( KICAD_INSTALL_DIR ${CMAKE_BINARY_DIR}/kicad-dest )
 set( KICAD_CMAKE_ARGS -DDEFAULT_INSTALL_PATH=/Library/Application\ Support/kicad -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOS_MIN_VERSION} -DwxWidgets_CONFIG_EXECUTABLE=${wxwidgets_INSTALL_DIR}/bin/wx-config  -DKICAD_SCRIPTING=ON  -DKICAD_SCRIPTING_MODULES=OFF  -DKICAD_SCRIPTING_WXPYTHON=OFF  -DPYTHON_EXECUTABLE=${PYTHON_INSTALL_DIR}/bin/python2.7  -DPYTHON_INCLUDE_DIR=${PYTHON_INSTALL_DIR}/include/python2.7/ -DPYTHON_LIBRARY=${PYTHON_INSTALL_DIR}/lib/libpython2.7.dylib  -DPYTHON_SITE_PACKAGE_PATH=${wxwidgets_INSTALL_DIR}/lib/python2.7/site-packages -DCMAKE_INSTALL_PREFIX=${KICAD_INSTALL_DIR} -DKICAD_SPICE=ON -DKICAD_USE_OCE=ON -DCMAKE_BUILD_TYPE=Release )
 
 # find OCE
-execute_process(COMMAND find /usr/local/Cellar/oce -name Resources OUTPUT_VARIABLE OCE_DIR RESULT_VARIABLE FOUND_OCE)
-
-if (NOT($(FOUND_OCE) EQUAL 0))
+execute_process(COMMAND find /usr/local/Cellar/oce -name Resources OUTPUT_VARIABLE OCE_DIR RESULT_VARIABLE FOUND_OCE_EXIT_CODE)
```
Comment (nickoe) [inline kicad-mac-builder/CMakeLists.txt]: You probably want to do something similar to what was done in the old scripts here:
https://github.com/wayneandlayne/KiCadMacOSPackaging/commit/9ec546904b93c69475d637ae69e00e35a26f02ca

**Comment by adamwolf** [comment]
Yup, that's a great idea.  It's next on my list after I finish up any
regressions from the Python Framework stuff I spent the weekend on.

**Comment by adamwolf** [comment]
@nickoe can you trigger a build for this one too?  thanks!

**Comment by nickoe** [comment]
http://ci.kicad-pcb.org/job/osx-kicad-adam-new-head/104/

**Comment by adamwolf** [comment]
🤞

**Comment by adamwolf** [comment]
Alright, since it built, any idea how to test if it's working? :)

**Comment by nickoe** [comment]
You just need to add a step or iges model to a  footprint and view it in any 3d view window.

**Comment by nickoe** [comment]
http://downloads.kicad-pcb.org/osx/testing/kicad-unified-20180427-112652-3f5f591.dmg

**Comment by adamwolf** [comment]
Is there a STEP or IGES model in a footprint that we already downloaded?
Can we check if a particular footprint is shown and then we know this works?

**Pull request merged**